src/test/lambda/commands/uploadLamda.test.ts

```diff
@@ -0,0 +1,61 @@
+/*!
+ * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import * as vscode from 'vscode'
+import * as assert from 'assert'
+import * as fs from 'fs-extra'
+import * as path from 'path'
+import { makeTemporaryToolkitFolder } from '../../../shared/filesystemUtilities'
+import { findApplicationJsonFile, getLambdaNamesFromApplicationJson } from '../../../lambda/commands/uploadLambda'
+
+describe('uploadLambda', async function () {
+    let tempFolder: string
+    let folderUri: vscode.Uri
+    const dotApplicationJsonData = `{
+        "DeploymentMethod": "lambda",
+        "Functions": {
+            "sampleFunction": {
+                "PhysicalId": {
+                    "us-west-2": "sampleFunction"
+                }
+            }
+        }
+    }`
+    beforeEach(async function () {
+        tempFolder = await makeTemporaryToolkitFolder()
+        folderUri = vscode.Uri.parse(tempFolder)
```

Comment:
Don't use `Uri.parse` for FS paths. Use `Uri.file`.
`parse` is only to be used when you have a URI as a string. 
Also, use `Uri.joinPath` if you already have a URI instead of `path.join`.